recipes/alibi/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "alibi" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/alibi-{{ version }}.tar.gz
+  sha256: 20131a369871b32bc7abfe1caa47131657abc0e41c7f5c8e0d2cc6105a816a96
+
+build:
+  number: 0
```

Comment:
`noarch: python` ?

Reply:
Are you wondering if this could be made a noarch package?

Reply:
Okay. Just tested with `noarch: python` and it went fine. Please the results here:

- https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=444410&view=results

I had some lines commented temporarily. Will update/drop them now.

Reply:
Done.